test/aqua/operators/test_abelian_grouper.py

```diff
@@ -26,7 +26,7 @@ class TestAbelianGrouper(QiskitAquaTestCase):
     """Abelian Grouper tests."""
 
     def test_abelian_grouper(self):
```

Comment:
`test_abelian_grouper` and `test_abelian_grouper2` are the same except they test different `paulis`. Could you combine them into one? For this you can use the `ddt` package of Python, e.g. like this:

```python
from ddt import ddt, data

@ddt
class TestAbelianGrouper(...):
    ...
    @data('h2_op', 'generic'):
    def test_commutation(self, pauli_op):
        if pauli_op == 'h2_op':
            paulis = (-1.05... )
        else:
            paulis = ( I ^ I ^ I ... )
```

Reply:
OK